codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/alltypesrequest.java

```diff
@@ -538,178 +539,104 @@ public boolean equals(Object obj) {
-        }
-        if (mapOfStringToIntegerList() != null) {
-            sb.append("MapOfStringToIntegerList: ").append(mapOfStringToIntegerList()).append(",");
-        }
-        if (mapOfStringToString() != null) {
-            sb.append("MapOfStringToString: ").append(mapOfStringToString()).append(",");
-        }
-        if (mapOfStringToSimpleStruct() != null) {
-            sb.append("MapOfStringToSimpleStruct: ").append(mapOfStringToSimpleStruct()).append(",");
-        }
-        if (mapOfEnumToEnumStrings() != null) {
-            sb.append("MapOfEnumToEnum: ").append(mapOfEnumToEnumStrings()).append(",");
-        }
-        if (mapOfEnumToStringStrings() != null) {
-            sb.append("MapOfEnumToString: ").append(mapOfEnumToStringStrings()).append(",");
-        }
-        if (mapOfStringToEnumStrings() != null) {
-            sb.append("MapOfStringToEnum: ").append(mapOfStringToEnumStrings()).append(",");
-        }
-        if (mapOfEnumToSimpleStructStrings() != null) {
-            sb.append("MapOfEnumToSimpleStruct: ").append(mapOfEnumToSimpleStructStrings()).append(",");
-        }
-        if (timestampMember() != null) {
-            sb.append("TimestampMember: ").append(timestampMember()).append(",");
-        }
-        if (structWithNestedTimestampMember() != null) {
-            sb.append("StructWithNestedTimestampMember: ").append(structWithNestedTimestampMember()).append(",");
-        }
-        if (blobArg() != null) {
-            sb.append("BlobArg: ").append(blobArg()).append(",");
-        }
-        if (structWithNestedBlob() != null) {
-            sb.append("StructWithNestedBlob: ").append(structWithNestedBlob()).append(",");
-        }
-        if (blobMap() != null) {
-            sb.append("BlobMap: ").append(blobMap()).append(",");
-        }
-        if (listOfBlobs() != null) {
-            sb.append("ListOfBlobs: ").append(listOfBlobs()).append(",");
-        }
-        if (recursiveStruct() != null) {
-            sb.append("RecursiveStruct: ").append(recursiveStruct()).append(",");
-        }
-        if (polymorphicTypeWithSubTypes() != null) {
-            sb.append("PolymorphicTypeWithSubTypes: ").append(polymorphicTypeWithSubTypes()).append(",");
-        }
-        if (polymorphicTypeWithoutSubTypes() != null) {
-            sb.append("PolymorphicTypeWithoutSubTypes: ").append(polymorphicTypeWithoutSubTypes()).append(",");
-        }
-        if (enumTypeString() != null) {
-            sb.append("EnumType: ").append(enumTypeString()).append(",");
-        }
-        if (sb.length() > 1) {
-            sb.setLength(sb.length() - 1);
-        }
-        sb.append("}");
-        return sb.toString();
+        return ToStringResult.builder("AllTypesRequest").add("StringMember", stringMember())
+                             .add("IntegerMember", integerMember()).add("BooleanMember", booleanMember()).add("FloatMember", floatMember())
```

Comment:
Rather have these all on new lines but guess not going to work that way in the formatter...

Reply:
Yeah, i tried adding a manual newline between each line, but the auto-formatter removed them. I'm not sure if it's worth the effort to try to fix, since it may make other stuff ugly to tell the formatter to preserve newlines.